nebula/ui/components/VPNServerList.qml

```diff
@@ -131,7 +131,7 @@ FocusScope {
                 anchors.right: parent.right
                 anchors.rightMargin: VPNTheme.theme.windowMargin / 2
                 anchors.leftMargin: anchors.rightMargin
-                visible: showRecentConnections && serverSearchInput.text.length === 0
+                visible: showRecentConnections && serverSearchInput.text.length === 0 && !tutorialUI.visible // TODO (unhide recent connections during tutorials)
```

Comment:
Can you tell me more about this TODO? Is it a missing thing in the tutorial logic?

Reply:
Gone :)